index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dienebi Ombu - Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav class="toolbar">
+            <a class="logo" href="index.html">ebidien</a>
+            <div class="main-menu">
+                <img src="menu-button.svg" alt="Main Menu">
+            </div>
+        </nav>
+    </header>
+    <section class="headline">
+        <div>
+            <div class="intro-title">
+                <h1>I'm Dienebi. Glad to see you!</h1>
+            </div>
+            <div class="intro-body">
+                <p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <h2>LET'S CONNECT</h2>
+            </div>
+            <ul class="social-media-icons">
+                <li class="social-media-icon"><a href="https://twitter.com/ebidien83"><img src="twitter.png"
+                            alt="twitter" /></a></li>
+                <li class="social-media-icon"><a href="https://www.linkedin.com/in/ebidien"><img src="linkedin.png"
+                            alt="linkedin" /></a></li>
+                <li class="social-media-icon"><a href="https://medium.com/@ebidien"><img src="medium.png"
+                            alt="medium" /></a></li>
+                <li class="social-media-icon"><a href="https://github.com/ebidien"><img src="github.png"
+                            alt="github" /></a></li>
+                <li class="social-media-icon"><a href="https://angel.co/u/dienebi-ombu"><img src="angellist.png"
+                            alt="angellist" /></a></li>
+            </ul>
```

Comment:
- [OPTIONAL ] I think it would be better to avoid unnecessary Divs as they are overloading your project making it unclean :thinking:

Reply:
Okay @GedeonTS, thanks. I will apply this subsequently as I get a better understanding of the relationship between divs and flexbox..